src/components/SearchPage.vue

```diff
@@ -94,7 +101,18 @@ export default {
     selectedSystemIds() {
       const { selectedSystems } = this;
       return selectedSystems.map(system => system.id);
+    },
+
+    selectedSystemLabels() {
+      const { selectedSystemIds } = this;
+      return selectedSystemIds.map(system => {
+        const foundSystem = bodySystems.find(s => {
+          return s.id === system;
+        });
+        return foundSystem.label;
+      });
```

Comment:
I'm probably missing something here, but it looks like could you could do this:

```javascript
selectedSystemLabels() {
  const { selectedSystems } = this;
  return selectedSystems.map(s => s.label);
}```

Reply:
It's complicated since the `selectedSystems` only contains an the following data structure: `[{id: HP:term}, ...]`. I didn't feel like restructuring the `selectedSystems` since that's what is persisted I believe, hence the more complicated approach.

Reply:
That's too bad. I suppose you could flip the logic instead:

```javascript
selectedSystemLabels() {
  const { selectedSystemIds } = this;
  return bodySystems.filter(s => selectedSystemIds.includes(s.id))
    .map(s => s.label);
}
```

Reply:
That's probably better. I wanted to start with the smaller set for reasons, but this is likely the better approach. I'll give it a shot.